bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/connection/KNXConnection.java

```diff
@@ -433,6 +440,14 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
                 }
             }
 
+            String shouldUseNAT = (String) config.get("useNAT");
+            if (StringUtils.isNotBlank(shouldUseNAT)) {
+                sUseNAT = shouldUseNAT.equalsIgnoreCase("true");
+            }
+            else if (StringUtils.isBlank(shouldUseNAT)) {
+                sLogger.warn("Error when trying to read parameter 'useNAT' from configuration. Parameter was missing, so using default.");
```

Comment:
If `useNat` is optional, this should be silent unless DEBUG level so the log doesn't have a WARN every time.